modules/apigee_m10n_add_credit/apigee_m10n_add_credit.module

```diff
@@ -107,16 +109,19 @@ function apigee_m10n_add_credit_commerce_product_access(EntityInterface $entity,
   return apigee_m10n_add_credit_service()->commerceProductAccess($entity, $operation, $account);
 }
 
+/**
+ * Implements hook_apigee_m10n_insufficient_balance_error_message_alter().
+ */
+function apigee_m10n_add_credit_apigee_m10n_insufficient_balance_error_message_alter(TranslatableMarkup &$message, SubscriptionInterface $subscription) {
+  return apigee_m10n_add_credit_service()->insufficientBalanceErrorMessageAlter($message, $subscription);
+}
+
 /**
  * Helper to get the `apigee_m10n.add_credit` service.
  *
  * @return \Drupal\apigee_m10n_add_credit\AddCreditServiceInterface
  *   The `apigee_m10n.add_credit` service.
  */
 function apigee_m10n_add_credit_service() {
```

Comment:
This helper is not doing much now. In a future PR, we could call `\Drupal::service('apigee_m10n.add_credit')` directly.